adminSiteClient/AdminLayout.tsx

```diff
@@ -22,32 +22,32 @@ export class AdminLayout extends React.Component<{
     static contextType = AdminAppContext
     context!: AdminAppContextType
 
-    @observable private showFAQ: boolean = false
-    @observable private showSidebar: boolean = false
+    @observable private showFAQ = false
+    @observable private showSidebar = false
```

Comment:
Why are these no longer explicit?

Reply:
This is an artifact of an attempt at automating this - I removed almost all of these again but some where still in there and if the file had some otherwise beneficial changes I left them in since I will tackle them soon anyhow

Reply:
:+1: Did you miss these on the 2nd pass or intending to fix later?

Reply:
I can see more of them – is it fine if I push a commit to revert the removal of types from initialized properties? Since I'm reviewing it all, it's fairly easy.